device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,498 @@
+{
+    "chassis": {
+        "name": "Mellanox-SN2700",
+        "model": "FIXME",
+        "serial": "FIXME",
+        "components": [],
+        "modules": [],
+        "fans": [
+            {
+                "name": "Fan 1",
+                "direction": "intake",
+                "tolerance": 20,
+                "led_status": "green"
```

Comment:
Remove all `led_status` fields, as this is not technically static data.

Reply:
fixed